testing/grid_ci.env

```diff
@@ -1,5 +1,6 @@
 BROWSER=chrome
 BROWSERSTACK=false
-HEADLESS=true
-HUB_URL=http://design-system-hub.test:4444
+HEADLESS=false
```

Comment:
Should this be true? I have a feeling this might be a value I changed whilst trying to debug the branch when we paired.

Reply:
No idea, Will set it back to be true. No issues really. Makes the tests slightly quicker so I suppose it makes sense.